### ISSUE TITLE
hlsdl v1.12.0 not available from haxelib

### ISSUE THREAD

**Issue by drflamemontgomery** [HaxeFoundation/hashlink#562]
haxelib doesn't have hlsdl v1.12.0 which breaks the latest version of heaps because of heaps reliance on v1.12.0
````
$ haxelib install hlsdl 1.12.0
Error: No such version 1.12.0 for library hlsdl
````
````
$ haxe -cp src -main Main -lib heaps -lib hlsdl
C:\HaxeToolkit\haxe\lib\heaps/git/hxd/Window.hl.hx:175: characters 37-46 : sdl.Window has no field warpMouse
C:\HaxeToolkit\haxe\lib\heaps/git/hxd/Window.hl.hx:265: characters 13-22 : sdl.Window has no field warpMouse
C:\HaxeToolkit\haxe\lib\heaps/git/hxd/Window.hl.hx:219: characters 17-21 : sdl.Window has no field grab
C:\HaxeToolkit\haxe\lib\heaps/git/hxd/Window.hl.hx:230: characters 17-21 : sdl.Window has no field grab
````

**Comment by tobil4sk** [comment]
@ncannasse It seems you are the only person with permissions to update these libs. Seems like it's as simple as:
```sh
git checkout 1.12
make HLIB=directx release_haxelib_package
make HLIB=sdl release_haxelib_package
# openal is not necessary to update as the code hasn't changed since 1.4
```

@drflamemontgomery If you are using the latest commit of heaps, then the 1.12 versions of these libs probably won't be compatible either. You probably have to use the latest commit of the hashlink libs as well, either by doing:
```sh
# if you DID want 1.12, you could change "master" for "1.12"
haxelib git hlsdl https://github.com/HaxeFoundation/hashlink master libs/sdl/
```
Or, to avoid having an entire clone of the hashlink repo for each of the hl libs:
```sh
# again, if you wanted 1.12 instead of the latest commit you could add "--branch 1.12"
git clone https://github.com/HaxeFoundation/hashlink
cd hashlink
haxelib dev hashlink other/haxelib
haxelib dev hlsdl libs/sdl
haxelib dev hldx libs/directx
# the version of hlopenal on haxelib is the newest version, so it's fine to install it from there
```

**Comment by Tekkunsan** [comment]
I tried this solution but it seems like it failed to load function

![image](https://user-images.githubusercontent.com/35391378/208295014-aad70454-0b3f-4faa-a37b-6a5039f23309.png)

any idea how to fix this?

**Comment by ncannasse** [comment]
I have released HLSDL and HLDX 1.13 (compatible with recently released HL 1.13)
